helper-cli/src/main/kotlin/commands/packagecuration/CreateCommand.kt

```diff
@@ -56,27 +57,32 @@ internal class CreateCommand : CliktCommand(
     override fun run() {
         val outputFile = getSplitCurationFile(outputDir, packageId, FileFormat.YAML.fileExtension)
 
-        if (outputFile.exists()) {
-            throw UsageError("File ${outputFile.absolutePath} already exists.")
+        val curations = if (outputFile.isFile) {
+            outputFile.readValue<MutableSet<PackageCuration>>()
+        } else {
+            mutableSetOf()
         }
 
+        if (packageId in curations.map { it.id }) {
```

Comment:
Why is it invalid to have multiple curations for a single `id`?

For example, the following could not be put anymore into separate curations for a single package.
* concluded license
* declared_license_mapping
* vcs info
* source artifact url

Reply:
That would be done in a single entry for one identifier, right?
```yml
- id: "Maven:com.foo:bar:1.0"
  curations:
    comment: "An explanation why the curation is needed or the reasoning for a license conclusion"
    concluded_license: "valid SPDX license expression to override the license findings."
    declared_licenses:
    - "license a"
    - "license b"
    description: "Curated description"
    homepage_url: "http://example.com"
    binary_artifact:
      url: "http://example.com/binary.zip"
      hash: "ddce269a1e3d054cae349621c198dd52"
      hash_algorithm: "MD5"
    source_artifact:
      url: "http://example.com/sources.zip"
      hash: "ddce269a1e3d054cae349621c198dd52"
      hash_algorithm: "MD5"
    vcs:
      type: "git"
      url: "http://example.com/repo.git"
      revision: "1234abc"
      path: "subdirectory"
    is_meta_data_only: true
    is_modified: true
    declared_license_mapping:
      "license a": "Apache-2.0"
```

And not by adding a separate entry for `- id: "Maven:com.foo:bar:1.0"` (This would cause issues in ORT, because the user wouldn't know which curation is applied).


The check is being done to avoid overwriting or additionally adding `- id: "Maven:com.foo:bar:1.0"` again to the same file.

Reply:
> That would be done in a single entry for one identifier, right?

I would prefer not to introduce the assumption the such curations are always done in a single entry.
Actually doing one thing at a time to me seems to be cleaner and has several advantages.
Just consider the `comment`. It has to provide a rationale for the curation. describing a single thing is much more straight forward compared to describing all things at once. It's a bit similar to "atomic commits" best practice IMO.

Reply:
Does ORT actually merge multiple curations for a single `package-id`?
What does that mean for this change? Should the command add a new curation with exactly the same package id?

Reply:
> Does ORT actually merge multiple curations for a single `package-id`?

Yes, somewhat: They are not merged, but applied in order, one after the other.

Reply:
> What does that mean for this change? Should the command add a new curation with exactly the same package id?

IMO it should at least not throw an exception if multiple curations with same identifier are in the file.

Whether it should always add a new curation: initially I was wondering whether it should add that template at all or just (create and) open that file. No strong opinion on that, I'd tend towards either always or never creating the template.

Reply:
> IMO it should at least not throw an exception if multiple curations with same identifier are in the file.

Yes I agree. I wasn't aware that ORT applies all curations for one `identifier`.

> I'd tend towards either always or never creating the template. What do you prefer?

In that case I would prefer to net creating the curation but printing the message that is as of now set as Exception message: `"Curation for ${packageId.toCoordinates()} already exists in '${outputFile.absolutePath}'."`

> open that file

That is not an option in the `helper-cli` but a wrapper script would do this, correct?

Reply:
> That is not an option in the helper-cli but a wrapper script would do this, correct?

Would agree.

Reply:
I removed the `Exception`. Please have another look.